complaints/ccdb/index_ccdb.py

```diff
@@ -81,8 +51,9 @@ def enhance_complaint(complaint):
                                 complaint.items()}
         # Restore complaint_what_happened to prevent ES queries from breaking
         normalized_complaint['complaint_what_happened'] = s
+        return normalized_complaint
 
-    return normalized_complaint
+    return complaint
```

Comment:
You may want to move the normalizing part outside of the `if qas_timestamp` block

Reply:
Great suggestion, I implemented this